examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb

```diff
@@ -0,0 +1,965 @@
+    "\n",
+    "**क्षेत्र:** ध्यान दें कि हम केवल डेटा के विकेंद्रीकृत (decentralized) / एन्क्रिप्ट (encrypt) करने के तरीके के बारे में बात नहीं कर रहे हैं, लेकिन हम यह बता रहे हैं कि कैसे डेटा के आसपास पूरे पारिस्थितिकी तंत्र को विकेंद्रीकृत करने में मदद करने के लिए PySyft का उपयोग किया जा सकता है, यहां तक कि डेटाबेस भी शामिल है जहां डेटा संग्रहीत और संगीन है, और तंत्रिका मॉडल (Neural Network) जिनका उपयोग डेटा से जानकारी निकालने के लिए किया जाता है। जैसा कि PySyft के लिए नए एक्सटेंशन बनाए गए हैं, नई कार्यक्षमता को समझाने के लिए इन नोटबुक को नए ट्यूटोरियल के साथ बढ़ाया जाएगा।\n",
+    "\n",
+    "लेखक:\n",
+    "- Andrew Trask - Twitter: [@iamtrask](https://twitter.com/iamtrask)\n",
+    "\n",
+    "अनुवादक:\n",
+    "- Arkadip Bhattacharya - Github: [@darkmatter18](https://github.com/darkmatter18)\n",
+    "\n",
+    "\n",
+    "\n",
+    "## रूपरेखा:\n",
+    "\n",
+    "- भाग 1: प्राइवेट डीप लर्निंग का बेसिक टूल\n",
+    "\n",
+    "\n",
+    "## यह ट्यूटोरियल क्यों लें?\n",
+    "\n",
+    "**1) प्रतियोगी कैरियर लाभ** - पिछले 20 वर्षों से, डिजिटल क्रांति ने बड़ी मात्रा में डेटा को अधिक से अधिक सुलभ बना दिया है क्योंकि एनालॉग प्रक्रियाएं डिजिटल हो गई हैं। \n",
+    "हालांकि, नए विनियमन जैसे कि [GDPR](https://eugdpr.org/) के साथ, उद्यमों पर दबाव होता है कि वे किस तरह से कम स्वतंत्रता का उपयोग करें - और अधिक महत्वपूर्ण बात यह है कि वे कैसे - व्यक्तिगत जानकारी का विश्लेषण करते हैं। **Bottom Line:** डेटा वैज्ञानिकों के पास \"पुराने स्कूल\"(Old school) टूल के साथ अधिक से अधिक डेटा तक पहुंच नहीं है, लेकिन प्राइवेट डीप लर्निंग के उपकरण सीखने से, आप इस वक्र से आगे हो सकते हैं और आपके करियर में प्रतिस्पर्धात्मक लाभ हो सकता है। \n",
+    "\n",
+    "**2) उद्यमी अवसर** - समाज में समस्याओं की एक पूरी मेजबानी है जिसे डीप लर्निंग हल कर सकता है, लेकिन कई सबसे महत्वपूर्ण का पता नहीं लगाया गया है क्योंकि इसे लोगों के बारे में अविश्वसनीय रूप से संवेदनशील जानकारी तक पहुंच की आवश्यकता होगी (मानसिक या रिश्ते के मुद्दों के साथ लोगों की मदद करने के लिए डीप लर्निंग का उपयोग करने पर विचार करें) !)। इस प्रकार, निजी डीप लर्निंग सीखना आपके लिए नए स्टार्टअप अवसरों की एक पूरी मेजबानी को अनलॉक करता है जो पहले इन टूलसेट के बिना दूसरों के लिए उपलब्ध नहीं थे।\n",
+    "\n",
+    "**3) सोशल गुड** - डीप लर्निंग का उपयोग वास्तविक दुनिया में कई प्रकार की समस्याओं को हल करने के लिए किया जा सकता है, लेकिन डीप लर्निंग व्यक्तिगत जानकारी पर डीप लर्निंग लोगों के बारे में, लोगों के लिए है। दीप लर्निंग कैसे करना है डेटा पर सीखना आपके पास कैरियर या उद्यमशीलता के अवसर से अधिक का प्रतिनिधित्व नहीं करता है, यह लोगों के जीवन में कुछ सबसे व्यक्तिगत और महत्वपूर्ण समस्याओं को हल करने में मदद करने का अवसर है - और इसे पैमाने पर करने के लिए।\n",
+    "\n",
+    "## मुझे अतिरिक्त क्रेडिट कैसे मिलेगा?\n",
+    "\n",
+    "- Pysyft को Github पर लाइक करें! - [https://github.com/OpenMined/PySyft](https://github.com/OpenMined/PySyft)\n",
```

Comment:
Like can be changed with star here for following through with proper terminology.